src/main/resources/db/migration/0300/V347__ReportMetricReference.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE accelerator.standard_metrics
+    DROP COLUMN reference,
+    ADD COLUMN reference INTEGER NOT NULL CHECK (reference >= 0),
+    ADD COLUMN sub_reference INTEGER CHECK (sub_reference >= 0),
+    ADD COLUMN sub_sub_reference INTEGER CHECK (sub_sub_reference >= 0);
```

Comment:
Even though product confirmed only 3, do we think it could ever become 4? E.g. `2.1.4.7`?